ui/src/components/Button/index.tsx

```diff
@@ -12,6 +12,7 @@ type CommonProps = {
     textSize?: string;
     padding?: string;
     children?: React.ReactNode;
+    childClasses?: string;
```

Comment:
There's a `children:` variant for this. You can use it in the className prop like this: `children:border-none` in order to style the button child nodes.

Reply:
That's handy, I have switched it to use that